resources/assets/sass/components/_row.scss

```diff
@@ -69,6 +87,13 @@
 
             @include font-smoothing;
 
+            font-size: $font-size-larger  - 2;
```

Comment:
`$font-size-larger  - 2` should be written with a single space on each side of the operator: `$font-size-larger - 2`